### PR TITLE
firefox, get your sh*t together pack

### PR DESCRIPTION
**Pull request by okias**
Allows compile of master git.

### CODE REVIEW

**Comment by okias** [comment]
I put on a top wayland use-flag, with these days developing wayland support. It's not top, but it's improving..

File: www-client/firefox/firefox-9999.ebuild [hunk 1]
```diff
@@ -31,9 +31,11 @@ ASM_DEPEND=">=dev-lang/yasm-1.1"
 RDEPEND="
 	>=dev-libs/nspr-4.13.1
 	>=dev-libs/nss-3.28.3
+	>=dev-util/cbindgen-0.6.4
```
Comment (pkulev) [inline www-client/firefox/firefox-9999.ebuild]: This doesn't build. Is there 0.6.3 intended?

**Review by pkulev — CHANGES_REQUESTED** [review]

File: www-client/firefox/firefox-9999.ebuild (hunk 1 above)
Comment (okias) [reply www-client/firefox/firefox-9999.ebuild]: sorry, typo. 0.6.3 is correct

Comment (pkulev) [reply www-client/firefox/firefox-9999.ebuild]: Ok, please fixup this and I'll test it soon.

Comment (okias) [reply www-client/firefox/firefox-9999.ebuild]: Fixed in new rebase-amend

**Comment by okias** [comment]
Since PIE is default for Firefox, --enable-pie flag is unknown option and stops firefox from building. Disabled.
Rest of commits is untouched

**Comment by okias** [comment]
Tested-by: David Heidelberg <david@ixit.cz>

Compiles and work properly.

**Review by pkulev — APPROVED** [review]

**Comment by pkulev** [comment]
Works, thank you!

**Pull request merged**